docs/infrastructure/eks-quickstart.md

```diff
@@ -117,7 +117,7 @@ Prepare your infrastructure for the Ingress Controller setup, which is performed
     | Subnet (public)  | `kubernetes.io/cluster/codacy-cluster` = `shared`<br/>`kubernetes.io/role/elb` = `1`          |
     | Subnet (private) | `kubernetes.io/cluster/codacy-cluster` = `shared`<br/>`kubernetes.io/role/internal-elb` = `1` |
 
-    For more information refer to the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html#vpc-tagging).
+    For more information refer to the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html#vpc-subnet-tagging).
```

Comment:
Yes, this is the correct link. The process changed, though. Previously this was a mandatory manual step, now EKS adds the tag when needed. See also the notice on that page.

Reply:
Yes, now it seems that EKS adds the correct tags automatically. But it's still a good idea to have instructions recommending that the users check if everything's OK. :+1: